README.md

```diff
@@ -52,6 +52,16 @@ Also check the config for other values, like your homeserver domain.
 
 ## How to get Slack app credentials
 
+The bridge can authenticate with Slack in two ways:
+1. using the officially supported bot/app tokens
+ - Pros: officially supported by Slack, nice UX if used in relay mode
+ - Cons: You need access permissions to add the app to the Slack workspace and in case of OAuth also to the bridge config.yaml file
+2. using your user token
+ - Pros: don't need any permissions or access
+ - Cons: against Slack's Terms of Service, can look confusing in relay mode
```

Comment:
This all renders as 4 different lists, which I doubt is what you intended. If you want lists inside a list, you need to indent these appropriately.